src/main/java/com/chargerrobotics/subsystems/ChomperSubsystem.java

```diff
@@ -37,15 +39,12 @@ public static ChomperSubsystem getInstance() {
   }
 
   public ChomperSubsystem() {
-    //chomperUpDown = new WPI_TalonSRX(Constants.chomperUpDown);
+    chomperUpDown = new WPI_TalonSRX(Constants.chomperUpDown);
+    chomperUpDown.configReverseLimitSwitchSource(LimitSwitchSource.RemoteTalonSRX, LimitSwitchNormal.NormallyOpen);
```

Comment:
Is the limit switch connected directly to the TalonSRX? If so, you need to use `LimitSwitchSource.FeedbackConnector` and not `RemoteTalonSRX`. It won't work in the current state anyways, because if the limit switch is on another TalonSRX, you also need to specify the CAN ID of the TalonSRX to read from.

Reply:
According to [Trello](https://trello.com/c/1fOwBxpU/71-connect-hood-position-index-switch-to-dio-on-roborio), it looks like the limit switch is going into one of the roboRIO's digital in/out ports.

Reply:
According to [Trello](https://trello.com/c/HqFLQnhy/70-connect-collector-position-index-switch-to-dio-on-roborio), it looks like we're now going to connect the limit switch to a digital in/out port on the roboRIO.